.github/workflows/check-quiche-release.yml

```diff
@@ -0,0 +1,37 @@
+name: Check Quiche release
+
+on: 
+  schedule:
+    # runs weekly
+    - cron: '0 0 * * 0'
+ 
+jobs:
+  get-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          
+      - name: Get Quiche Release
+        shell: bash
+        env:
+          GH_TOKEN: ${{ github.token }}        
```

Comment:
Isn't this ... ?

```suggestion
          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}        
```

Reply:
that's perfectly fine and equivalent https://docs.github.com/en/actions/learn-github-actions/contexts#github-context